### DISCUSSION TITLE
DragonFly Works Flawlessly

### DISCUSSION THREAD

**Discussion by Gio2241** [dragonflydb/dragonfly#2112] [Show and tell]
I use DragonFly with Dapr.io as Redis replacement for all cases.

**Comment by kostasrim** [comment]
@Gio2241 We are very glad :heart: :rocket:

**Comment by romange** [comment]
@Gio2241 glad to hear! Could you tell us about your use-cases?

**Comment by Gio2241** [comment]
Already using without issues:
State management: https://docs.dapr.io/developing-applications/building-blocks/state-management/state-management-overview/
Pub/Sub (streaming): https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-overview/

Plan to use this component as well:
Distributed lock: https://docs.dapr.io/developing-applications/building-blocks/distributed-lock/distributed-lock-api-overview/

Does DragonFly support Redlock?
https://redis.io/docs/manual/patterns/distributed-locks/

**Comment by romange** [comment]
Yes, Dragonfly supports Redlock.

On Thu, Nov 2, 2023 at 7:00 PM Gio2241 ***@***.***> wrote:

> Already using without issues:
> State management:
> https://docs.dapr.io/developing-applications/building-blocks/state-management/state-management-overview/
> Pub/Sub (streaming):
> https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-overview/
>
> Plan to use this component as well:
> Distributed lock:
> https://docs.dapr.io/developing-applications/building-blocks/distributed-lock/distributed-lock-api-overview/
>
> Does DragonFly support Redlock?
> https://redis.io/docs/manual/patterns/distributed-locks/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dragonflydb/dragonfly/discussions/2112#discussioncomment-7459220>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4BFCC4M6JOETZ7ACF5333YCPGTHAVCNFSM6AAAAAA62VH5XWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINJZGIZDA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Roman Gershman
CTO
---
*www.dragonflydb.io <http://www.dragonflydb.io>*

**Comment by worldsoup** [comment]
Hey @Gio2241 ! We would love to partner with you on a blog post about how you use Dragonfly with Dapr...I think other developers in our community would be really interested in learning more. Could you send me an email to discuss? nick@dragonflydb.io

Thank you for your support!
